package.json

```diff
@@ -8,7 +8,8 @@
     "dev:electron": "cross-env NODE_ENV=development webpack --config webpack/electron.webpack.js --mode development && electron .",
     "dev:react": "cross-env NODE_ENV=development webpack-dev-server --config webpack/react.webpack.js --mode development",
     "electron:serve": "wait-on http-get://localhost:4000/ && npm run dev:electron",
-    "test": "jest"
+    "lint": "eslint './**/*.tsx' './**/*.ts' './**/*.js'",
```

Comment:
The lint command should target only files inside `src` or `electron`, this way i think `dist` files would be linted too.

Reply:
The `dist` folder is ignored on the `.eslintignore` file.
Either way, I just pushed a commit with the command updated to `eslint --ext .tsx,.ts,.js src/ electron/` 🙂.